wezterm-gui/src/glyphcache.rs

```diff
@@ -4239,6 +4259,62 @@ impl<T: Texture2d> GlyphCache<T> {
                     );
                 }
             }
+            BlockKey::Braille(dots_pattern) => {
+                // `dots_pattern` is a byte whose bits corresponds to dots
+                // on a 2 by 4 dots-grid.
+                // The position of a dot for a bit position (1-indexed) is as follow:
+                // 1 4  |
+                // 2 5  |<- These 3 lines are filled first (for the first 64 symbols)
+                // 3 6  |
+                // 7 8  <- This last line is filled last (for the remaining 192 symbols)
+
+                let cell_width = self.metrics.cell_size.width as f32 / 2.;
+                let cell_height = self.metrics.cell_size.height as f32 / 4.;
+                let center_offset_x = cell_width / 2.;
```

Comment:
> Another thing to consider: for the 6-dot braille glyphs, it probably looks clearer to omit positions 7 and 8 entirely and then the vertical dot positioning has more space and can be based on quarters instead of fifths.

I think braille-based terminal drawing tools/libs will hate you for this :eyes:

Reply:
Ah, I didn't realize that they weren't distinct from the 8 dot variants.  I agree: it would be horrible to omit the bottom row in that case!